src/components/editor/property/InputListLOC.jsx

```diff
@@ -97,7 +104,7 @@ class InputListLOC extends Component {
           ref={typeahead => this.typeahead = typeahead}
           onFocus={() => {
             this.setState({ isLoading: true })
-            fetch(`${this.props.lookupConfig.uri}.json`)
+            fetch(`${this.props.lookupConfig[0].uri}.json`)
```

Comment:
previously, what was passed was the first one, so this represents moving the hardcoding from PropertyComponent to this component

Reply:
see comment above - this feels hinky but I let it slide.